package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@cda0/terrajs",
-  "version": "0.3.0",
-  "description": "A node interface to terraform",
+  "version": "0.3.1",
```

Comment:
Can we bump to 0.4 as it is a change to the interface (even though it is not breaking?)